kairos-deposit-contract-tests/src/test_fixture/utils.rs

```diff
@@ -0,0 +1,46 @@
+use casper_engine_test_support::{
+    ExecuteRequestBuilder, LmdbWasmTestBuilder, ARG_AMOUNT, DEFAULT_ACCOUNT_ADDR,
+    DEFAULT_ACCOUNT_INITIAL_BALANCE,
+};
+use casper_types::{
+    account::AccountHash,
+    runtime_args,
+    system::{handle_payment::ARG_TARGET, mint::ARG_ID},
+    PublicKey, SecretKey,
+};
+
+// Creates a dummy account and transfer funds to it
```

Comment:
Since this is the tests crate, the case where the account_string is None doesn't need to be handled. We just need a set of n dummy accounts to work with, there are no edgecases.

Reply:
That is exactly what I mean. I'm pointing out that these non-obvious conditions need to be documented, such that any developer using or extending this function understands the intention. I didn't know this, and I still dont understand what the `None` case does in case the account string is None.
If you say that the None case is impossible, then it would be better to design the interface that way or maybe use `expect`